code/Base/PartySystem/Party.cs

```diff
@@ -0,0 +1,105 @@
+namespace Sports.PartySystem;
+
+public partial class Party : Entity // Use Entity for Parties since BaseNetworkables don't like Lists currently
+{
+	[Net] private Client _partyHost { get; set; }
+	public Client Host
+	{
+		get => _partyHost; set
+		{
+			if ( _partyHost == null )
+				_partyHost = value;
+		}
+	}
+	[Net] public IList<Client> Members { get; set; } // Can't store the PartyComponent Crashes game when adding to list directly after creation
+
+	/// <summary>
+	/// kick a Client from the Party
+	/// </summary>
+	/// <param name="ClientNetworkID"></param>
+	[ServerCmd]
+	public static void KickPlayer( int ClientNetworkID )
+	{
+		if ( ConsoleSystem.Caller is not Client caller )
+			return;
+		var callerComp = caller.Components.Get<PartyComponent>();
+		if ( callerComp?.Party?.Host != caller )
+		{
+			Log.Debug( "Only the party host can kick players" );
+			Log.Debug( "Party host: " + callerComp?.Party?.Host?.Name );
+			return;
+		}
+		if ( callerComp?.Party?.Members.FirstOrDefault( e => e.NetworkIdent == ClientNetworkID )?.Components.Get<PartyComponent>( true ) is PartyComponent comp )
+		{
+			comp.Leave();
+		}
+	}
+	/// <summary>
+	/// Accept an Invite
+	/// </summary>
+	/// <param name="ClientNetworkID"></param>
+	[ServerCmd]
+	public static void AcceptInvite( int ClientNetworkID )
+	{
+		if ( ConsoleSystem.Caller is not Client caller )
+			return;
+		var callerComp = caller.Components.Get<PartyComponent>();
+		if ( Client.All.FirstOrDefault( e => e.NetworkIdent == ClientNetworkID )?.Components.Get<PartyComponent>( true ) is PartyComponent comp )
+		{
+			if ( comp.Party == null )
+			{
+				PartyManager.CreatePartyFor( comp );
+			}
+			callerComp.Party = comp.Party;
+		}
+	}
+
+	public override void Spawn()
+	{
+		base.Spawn();
+		Transmit = TransmitType.Always;
+	}
+	/// <summary>
```

Comment:
Spacing between this doc comment and the method above it should be fixed (here and quite a few other places).